distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -11,153 +11,88 @@
 
 == {title}
 
-The Solr Catalog Provider is included with a standard installation of ${branding}.
-The following are the three ways to configure the Solr Catalog Provider:
+The Solr Catalog Provider is included with a standard installation of ${branding}. There are several configurations available:
 
-<<_http_solr,HTTP Solr>> (default):: Used with an external (HTTP) Solr server.
+Internally Managed Solr Server (default):: ${branding} comes bundled with a customized distribution
+of Apache Solr. This Solr distribution includes the configuration files needed for
+the Solr cores and special JAR libraries used by ${branding}.
+This installation is referred to as _internally managed_
+because the ${branding} scripts attempt to start and stop this Solr server. The advantages of the
+internally managed Solr configuration is "unzip and go" -- no additional configuration is required.
```

Comment:
This line was a little confusing to me, maybe add a few words to help. Something like `managed Solr configuration is that it is ready to "unzip and go"`